src/pages/ActiveVotation.tsx

```diff
@@ -316,87 +324,94 @@ const Votation: React.FC = () => {
   }
 
   return (
-    <Center sx={outerContainer}>
-      {(castVoteLoading || blankVoteLoading || stvLoading) && <Loading text="Registrerer stemme" asOverlay={true} />}
-      <VStack sx={centerContainer} maxWidth="800px" alignItems="left" spacing="3em">
-        <VStack alignItems="left" spacing="0.5rem">
-          <Heading as="h1" style={subtitlesStyle}>
-            Votering {data.votationById.index + 1}
-          </Heading>
-          <Heading as="h1" sx={h1Style}>
-            {data.votationById.title}
-          </Heading>
-
-          <Text mt="1em" mb="2em">
-            {data.votationById.description}
-          </Text>
-        </VStack>
-
-        {status === VotationStatus.Open && (
-          <CastVote
-            alternatives={alternatives || []}
-            handleSelect={handleSelect}
-            blankVotes={data.votationById.blankVotes || false}
-            submitVote={submitVote}
-            submitButtonDisabled={selectedAlternativeId === null && data.votationById.type !== VotationType.Stv}
-            voteCount={voteCount}
-            votingEligibleCount={data?.getVoteCount?.votingEligibleCount}
-            isStv={data.votationById.type === VotationType.Stv}
-            updateAlternatives={setAlternatives}
-            userHasVoted={userHasVoted}
-            showVote={showVote}
-            isVotingEligible={isVotingEligible}
-          />
-        )}
-        {status === VotationStatus.CheckingResult && participantRole === Role.Participant && (
-          <Box>
-            <Loading asOverlay={false} text={'Resultatene sjekkes'} />
-          </Box>
-        )}
-        {status === VotationStatus.CheckingResult &&
-          (participantRole === Role.Counter || participantRole === Role.Admin) && (
-            <CheckResults
-              result={votationResultData}
-              isStv={data.votationById.type === VotationType.Stv}
-              role={participantRole}
-              votationId={votationId}
-              meetingId={meetingId}
-            />
+    <>
+      <Box bg={offwhite} w="100vw" color="gray.500" style={styles}>
```

Comment:
`<HStack bg={offwhite} w="100vw" color="grey.500" alignItems="center">` would do the same